docs/javascript/src/challenges/Dictionary App/script.js

```diff
@@ -0,0 +1,48 @@
+const input = document.querySelector('.search-input')
+const searchBtn = document.querySelector('.search-btn')
+const word_searched = document.querySelector('.word')
+const partOfSpeech = document.querySelector('.part-of-speech')
+const phonetic = document.querySelector('.phonetic')
+const meaning = document.querySelector('.meaning')
+const sentence = document.querySelector('.sentence')
+const soundBtn = document.querySelector('.sound-btn')
+const wordBody = document.querySelector('.content-body')
+const notFound = document.querySelector('.not-found')
+const audio = document.querySelector('audio')
+
+// fetch data from api and set in our project
+showData = async (word) => {
+    const url = `https://api.dictionaryapi.dev/api/v2/entries/en/${word}`
+    const response = await fetch(url)
+    const word_data = await response.json()

```

Comment:
plz move the fetching part to a different function & also take care of error handling.